docs/PLUGINS.md

```diff
@@ -20,6 +20,5 @@ If you just want to get some plugins for your Discourse instance, check out [the
 ### Simple Demonstration Plugins
 
 - [Manipulating text in the composer](https://meta.discourse.org/t/plugin-tutorial-1-how-to-manipulate-the-text-in-the-composer/10925)
- 
-- [Display plugin outlet locations](https://meta.discourse.org/t/plugin-outlet-locations/29589)
 
+- [Display plugin outlet locations](https://meta.discourse.org/t/plugin-outlet-locations/29589)
```

Comment:
Weird... not sure what happened here... maybe a line-ending (CR vs. CRLF) normalization?